shared/src/arguments.rs

```diff
@@ -61,6 +64,14 @@ pub struct Arguments {
         use_delimiter = true
     )]
     pub baseline_sources: Vec<BaselineSources>,
+
+    /// The number of blocks kept in the pool cache.
+    #[structopt(long, env, default_value = "10")]
```

Comment:
Not sure what the best value is, but maybe 4 blocks (~1min on average) would be a good compromise. If we choose this value too high, we might be quoting outdated prices occasionally.

Reply:
We probably want a slightly higher value in the driver so we're sure we still have the pool state when we simulate solver solutions.
On the other hand in the order book as you say this is a bound on how outdated a price can be so it can be lower. Although as long as the token is in the LRU it will have updated pools anyway.